server/src/middleware/Authorization.js

```
@@ -0,0 +1,27 @@
+const jwt = require("jsonwebtoken");
+
+const verifyToken = (req, res, next) => {
+  if (req.headers && req.headers.authorization) {
```

can use fail fast approach here